src/gmail.js

```diff
@@ -2378,7 +2374,7 @@ var Gmail = function(localJQuery) {
 
         // loop through applicable types
         var types = type ? [ type ] : [ "before", "on", "after", "dom" ];
-        $.each(types, function(idx, type) {
+        types.forEach(type => {
```

Comment:
If we are replacing `$.each` with something else, I would rather we replace it with a normal Javascript `for..of`-iterator if possible.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
Yep, will change.

Reply:
Changed.